data.sql

```diff
@@ -7,3 +7,14 @@ VALUES
 ('Gabumon', '2018-11-15', 2, TRUE, 8.0),
 ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
 ('Devimon', '2017-05-12', 5, TRUE, 11.0);
+
+INSERT INTO animals
+(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '2020-02-08', 0, FALSE, -11),
+('Plantmon', '2021-11-15', 2, TRUE, -5.7),
+('Squirtle', '1993-04-02', 3, FALSE, -12.13),
+('Angemon', '2005-06-12', 1, TRUE, -45),
+('Boarmon', '2005-07-12', 7, TRUE, 20.4),
+('Blossom', '2005-10-13', 3, TRUE, 17),
```

Comment:
- [ ]  I believe you made a typo on the date of birth of this animal. The correct date should be `1998-10-13`